syntax = "proto3";

package authentication;

service AuthService {
    rpc getSampleData(SampleRequest) returns (SampleResponse);
}

message SampleRequest {
    string my_request = 1;
    string my_req_data = 2;
}

message SampleResponse {
    string my_response = 1;
    string my_resp_data = 2;
}

